src/components/DeckMechanisms/index.js

```diff
@@ -550,6 +560,9 @@ export default class DeckMechanisms extends React.Component {
       // Increment the current turn by 1
       newState.turn += 1
 
+      // Log the end turn action
+      newState.playedCards = [END_TURN_CARD, ...state.playedCards]
```

Comment:
```suggestion
      newState.playedCards.unshift(END_TURN_CARD)
```

Both are similar, so pick whatever you prefer. I’m fine with you.

Reply:
I had used unshift initially but I think I'll leave it this way for consistency with the rest